portal-api/src/v1/users/routes.js

```diff
@@ -154,32 +155,67 @@ module.exports.findById = (req, res, next) => {
 };
 
 module.exports.updateById = (req, res, next) => {
-  findOne(req.params._id, (error, user) => {
-    if (error) {
-      next(error);
-    } else if (user) {
-      const errors = applyUpdateRules(user, req.body);
-      if (errors.length) {
-        res.status(400).json({
-          success: false,
-          errors: {
-            count: errors.length,
-            errorList: combineErrors(errors),
-          },
-        });
+  if (req.user.roles.includes(ADMIN)) {
+    findOne(req.params._id, (error, user) => {
+      if (error) {
+        next(error);
+      } else if (user) {
```

Comment:
This could be changed to a simple guard like `if (!user) { [do something] }` to avoid the nesting below